plugins/woocommerce/includes/admin/class-wc-admin-notices.php

```diff
@@ -673,7 +673,7 @@ public static function email_sender_options_notice() {
 			 */
 			$notice_dismissed = apply_filters(
 				'woocommerce_hide_email_sender_options_notice',
-				get_user_meta( get_current_user_id(), 'dismissed_block_email_editor_notice', true )
+				get_user_meta( get_current_user_id(), 'dismissed_email_sender_options_notice', true )
```

Comment:
I missed this in the first review. I see that in the newer code in the [src](https://github.com/search?q=repo%3Awoocommerce%2Fwoocommerce+get_user_meta+path%3A%2F%5Eplugins%5C%2Fwoocommerce%5C%2Fsrc%5C%2F%2F+&type=code) the user meta is often prefixed `woocommerce_`. Perhaps we could do that as well so that it is less conflict-prone.